aces/Loris
# [src/Middleware/UserPageDecorationMiddleware.php] trying to get property of non object

## Brief summary of changes

Fixes a bug that pollutes the error_log when visiting a candidate profile.

Error output:
`[Mon Feb 03 12:53:30.699118 2020] [php7:notice] [pid 23155] [client ::1:52103] PHP Notice:  Trying to get property 'page' of non-object in /Users/alizee/Development/GitHub/McGill/Loris/src/Middleware/UserPageDecorationMiddleware.php on line 137, referer: http://localhost/candidate_parameters/?candID=300005&identifier=300005`

#### Testing instructions (if applicable)

1. Visit a candidate profile
2. Check the error_log
